app/models/katello/errata_status.rb

```diff
@@ -53,7 +69,7 @@ def to_status(_options = {})
         NEEDED_SECURITY_ERRATA
       elsif errata.any?
         NEEDED_ERRATA
-      elsif host.content_facet.bound_repositories.empty?
+      elsif host.content_facet.bound_repositories.empty? || host.installed_packages.empty?
```

Comment:
I could still see this being split out into separate messages for 
- missing package data
- missing repositories data

Reply:
oh yes, i mentioned that above, i think its valuable to do that!